Projects/CoX/Common/GameData/power_serializers.cpp

```
@@ -242,36 +242,42 @@ bool loadFrom(BinStore *s, AllPowerCategories &target)
-    archive(cereal::make_nvp("AllowResistance",src.AllowResistance)); // T/F
-    archive(cereal::make_nvp("StackType",src.StackType));
-    archive(cereal::make_nvp("Duration",src.Duration));
-    archive(cereal::make_nvp("Magnitude",src.Magnitude));
-    archive(cereal::make_nvp("ContinuingBits",src.ContinuingBits));
-    archive(cereal::make_nvp("ContinuingFX",src.ContinuingFX));
-    archive(cereal::make_nvp("ConditionalBits",src.ConditionalBits));
-    archive(cereal::make_nvp("ConditionalFX",src.ConditionalFX));
-    archive(cereal::make_nvp("EntityDef",src.EntityDef));
-    archive(cereal::make_nvp("PriorityListDefense",src.PriorityListDefense));
-    archive(cereal::make_nvp("PriorityListOffense",src.PriorityListOffense));
-    archive(cereal::make_nvp("PriorityListPassive",src.PriorityListPassive));
+        archive(cereal::make_nvp("CancelOnMiss",src.CancelOnMiss));     // T/F
+        archive(cereal::make_nvp("NearGround",src.NearGround));       // T/F
+        archive(cereal::make_nvp("AllowStrength",src.AllowStrength));    // T/F
+        archive(cereal::make_nvp("AllowResistance",src.AllowResistance)); // T/F
+        archive(cereal::make_nvp("StackType",src.StackType));
+        archive(cereal::make_nvp("Duration",src.Duration));
+        archive(cereal::make_nvp("Magnitude",src.Magnitude));
+        archive(cereal::make_nvp("ContinuingBits",src.ContinuingBits));
+        archive(cereal::make_nvp("ContinuingFX",src.ContinuingFX));
+        archive(cereal::make_nvp("ConditionalBits",src.ConditionalBits));
+        archive(cereal::make_nvp("ConditionalFX",src.ConditionalFX));
+        archive(cereal::make_nvp("EntityDef",src.EntityDef));
+        archive(cereal::make_nvp("PriorityListDefense",src.PriorityListDefense));
+        archive(cereal::make_nvp("PriorityListOffense",src.PriorityListOffense));
+        archive(cereal::make_nvp("PriorityListPassive",src.PriorityListPassive));
+    }
+    catch(cereal::Exception& e)
+    {
```

Does cereal already spit out an error message here, or should we display one?

Perhaps add a comment inside the block so it's clear why we aren't handling the catch.

---

I'm still testing what proper format is needed in the json, without a catch here it errors and doesn't let the server start. I was hoping to separate out name, scale, and maybe a few others as essential, so it would only ignore errors for data that can be ignored. But that didn't work so it will need a better check of validity.

---

Turns out, I  was getting an error from trying to assign a enum from text, when  it expexts a number. From googling, you can read text in a json into an enum if you do something like:
Qbytearray stacktype;
archive(cereal::make_nvp("StackType",stacktype));
If (stacktype == "replace")
Src.StackType = StackType::Replace;
Which seems a bit akward, but we could check for more variables this way, which makes the issue24 data easier to use. I think I'll have hold this back for a PR devoted to loading data.